bark/runtime/scenario/interaction_dataset_processing/dataset_decomposer.py

```diff
@@ -45,7 +46,7 @@ def __find_first_ts_on_map__(self, id_ego):
             for lane in lane_list:
                 polygon = self._map_interface.GetRoadgraph().GetLanePolygonForLaneId(lane.lane_id)
                 if Collide(polygon, point_agent):
-                    time_ego_first = state[0]*1e3  # use timestamp in ms
+                    time_ego_first = state[0]*1e3 + self._starting_offset_ms # use timestamp in ms
```

Comment:
is this not dataset specific? Could introduce a parameter for ms/s

Reply:
sure, I introduced a parameter

Reply:
But also the 1e3?